pwiz_tools/Skyline/Test/CodeInspectionTest.cs

```diff
@@ -215,6 +219,50 @@ List<string> CheckFormsWithoutTestRunnerLookups()
             return missing;
         }
 
+        /// <summary>
+        /// We have some code, especially in commandline, that expects certain error messages to start with "Error:" or the L10N equivalent
+        /// </summary>
+        void InspectConsistentErrorMessages(List<string> errors)
+        {
+            var currentCulture = Thread.CurrentThread.CurrentUICulture;
+            try
+            {
+                Thread.CurrentThread.CurrentCulture = Thread.CurrentThread.CurrentUICulture = new CultureInfo("en"); // We want to compare against the "en" resources
+                var resourceSet = Skyline.Properties.Resources.ResourceManager.GetResourceSet(Thread.CurrentThread.CurrentUICulture, true, true);
+
+                // Build the string of resources.
+                foreach (var resource in resourceSet)
+                {
+                    var pair = resource as DictionaryEntry? ?? default;
```

Comment:
In Visual Studio 2017, ReSharper does not like this "default" here and wants it to be "default(DictionaryEntry)".
Are we still supporting VS2017? Is there a setting I could change to make that error warning go away?

Reply:
I'll just replace it with "new DictionaryEntry()" for max portability.